content/index.fr.md

```diff
@@ -181,60 +181,60 @@ _Cette vidéo est aussi disponible sur un serveur du W3C : [Vidéo : Évaluer l'
     <tr>
-      <td>"L'outil de création de rapports WCAG-EM" vous permet d'enregistrer les résultats à mesure que vous suivez la méthodologie.</td>
-      <td>Outil de création de rapports WCAG-EM. Une barre de progression montre les différentes étapes de la méthodologie d'évaluation.</td>
+      <td>« L’outil de création de rapports WCAG-EM » vous permet d’enregistrer les résultats à mesure que vous suivez la méthodologie.</td>
+      <td>Outil de création de rapports WCAG-EM. Une barre de progression montre les différentes étapes de la méthodologie d’évaluation.</td>
     </tr>
     <tr>
-      <td>Pour vous aider à mieux comprendre comment les différents utilisateurs naviguent sur votre site Web, WAI fournit des conseils dans "Impliquer les utilisateurs dans les projets de développement Web pour une accessibilité plus optimale et plus facile" et "Impliquer les utilisateurs dans l'évaluation de l'accessibilité Web".</td>
-      <td>L'écran se divise en 12 parties sur chacune desquelles un utilisateur différent est face à un ordinateur. Impliquer des utilisateurs dans les projets Web pour une accessibilité plus optimale et plus facile et Impliquer des utilisateurs dans l'évaluation de l'accessibilité Web.</td>
+      <td>Pour vous aider à mieux comprendre comment les différents utilisateurs naviguent sur votre site Web, WAI fournit des conseils dans « Impliquer les utilisateurs dans les projets de développement Web pour une accessibilité plus optimale et plus facile » et « Impliquer les utilisateurs dans l’évaluation de l’accessibilité Web ».</td>
+      <td>L’écran se divise en 12 parties sur chacune desquelles un utilisateur différent est face à un ordinateur. Impliquer des utilisateurs dans les projets Web pour une accessibilité plus optimale et plus facile et Impliquer des utilisateurs dans l’évaluation de l’accessibilité Web.</td>
     </tr>
     <tr>
-      <td>Grâce à toutes ces ressources, vous pouvez apprendre comment vérifier l'accessibilité de votre site Web et donner la priorité aux problèmes que vous devez régler en premier.</td>
-      <td>Une loupe qui montre les mots "vérification de l'accessibilité" est remplacée par une liste de problèmes.</td>
+      <td>Grâce à toutes ces ressources, vous pouvez apprendre comment vérifier l’accessibilité de votre site Web et donner la priorité aux problèmes que vous devez régler en premier.</td>
+      <td>Une loupe qui montre les mots « vérification de l’accessibilité » est remplacée par une liste de problèmes.</td>
     </tr>
     <tr>
-      <td>Accessibilité Web : essentielle pour certains, utile à tous.</td>
-      <td>Des icônes autour d'un ordinateur s'affichent : une main ; un œil ; un cerveau ; une oreille ; et une bouche avec des ondes sonores.</td>
+      <td>Accessibilité Web : essentielle pour certains, utile à tous.</td>
+      <td>Des icônes autour d’un ordinateur s’affichent : une main ; un œil ; un cerveau ; une oreille ; et une bouche avec des ondes sonores.</td>
```

Comment:
@3l3gant-cod3s A non-breaking space is needed before each semi-colon, don't you think ?

Reply:
Hi Rémi,
There is in fact a (not narrow) non-breaking space that I typed on my Azerty+ keyboard (using AltGr+Space [http://norme-azerty.fr/img/main_layout.png](http://norme-azerty.fr/img/main_layout.png) ). That's why there's a marked difference between « s’affichent » and « : » in the above diff (as for « ' » and « ’ »).
You can copy-paste the diff in LibreOffice (or Word) and you will be able to visually perceive the difference (small grey rectangle instead of a white one) if you activate « Nonprinting Characters on/off [Ctrl+F10] / Activer les marques de formatage [ ¶ ] ». That same option shows/hides newlines too.
You probably mean an explicit (narrow) non-breaking space &#8239; ? It’s equivalent but I will follow whatever rule is in force. Or I can type a NNBS (narrow) like in « test » (instead of « test » if you can spot the difference, both non-breaking).
I should have done this before the semicolons too… :(

Reply:
I've done it for the semicolons too… :)